### PR TITLE
Added new command details

### PR DESCRIPTION
**Pull request by cs-shivaji-kolape**
Following changes added in this PR
1. Adds New `availableCommands` details
2. Updated help message screenshot. 
3. Updated `Create Alert` and Create Indictor Playbook Description.

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ You can create indicators on the fly for enrichment and further investigation by
 
 For more information and examples for this command, refer to the [Adding an indicator with the indicator value specified](https://docs.fortinet.com/document/fortisoar/1.0.0/fortisoar-for-microsoft-teams-application/630/fortisoar-for-microsoft-teams-application-v1-0-0#Adding_an_indicator_with_the_indicator_value_specified) section of FortiSOAR&trade; product documentation.
 
+## `@fortiSOAR availableCommands`
+
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:47]: This command brings up lists all available tags that can be used to trigger a playbook. --> This command brings up list of all available tags that can be used to trigger a playbook in FortiSOAR.

File: docs/res/help-message.png
Comment (cs-suyog-jinturkar) [inline docs/res/help-message.png:1]: Please review and change "Lists all available tag that...." to "List all available tags that...."

**Review by cs-suyog-jinturkar — APPROVED** [review]

File: docs/contents.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ When creating notification rules, selecting **Manual Input** as a notification t
 
 | Playbook Name          | Description                                                                                                                                   |
 |:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
-| Create Indicator       | Extracts indicators from the text provided as input                                                                                           |
-| Create Alert           | Creates an alert based on the values provided in the form                                                                                     |
+| Create Indicator       | Extracts indicators from the text provided as input. Gets triggered on createIndicator command when used in Microsoft Teams                   |
```
Comment (cs-abhishek-shukla) [inline docs/contents.md]: Extracts indicators from the text sent on the chat interface and is triggered when `createIndicator` command is sent through Microsoft Teams.

File: docs/contents.md [hunk 3]
```diff
@@ -47,8 +47,8 @@ When creating notification rules, selecting **Manual Input** as a notification t
...
+| Create Alert           | Creates an alert based on the values provided in the form. Gets triggered on createAlert command when used in Microsoft Teams                 |
```
Comment (cs-abhishek-shukla) [inline docs/contents.md]: Creates an alert based on the values provided in the form and is triggered when `createAlert` command is sent through Microsoft Teams.

File: docs/usage.md [hunk 4]
```diff
@@ -43,6 +43,13 @@ You can create indicators on the fly for enrichment and further investigation by
...
+This command brings up list of all available tags that can be used to trigger a playbook in FortiSOAR.
```
Comment (cs-abhishek-shukla) [inline docs/usage.md]: This command brings up a list of all available tags that can be used to trigger a playbook in FortiSOAR&trade;.

**Review by cs-abhishek-shukla — CHANGES_REQUESTED** [review]

**Review by cs-abhishek-shukla — APPROVED** [review]

**Pull request merged**